herald/src/app/+guild-profile/shared/guild-profile.service.ts

```diff
@@ -22,8 +22,9 @@ export class GuildProfileService {
   }
 
   private getGuildProfileFromFile(name: string) {
+    console.log('making call from guild profile service for ' + name);
     let fileName = name.replace(/ /g, '-');
-    return this.http.get(`/assets/data/guilds/${fileName}.json`)
+    return this.http.get(`https://www2.uthgard.net/herald/api/guild/${fileName}`)
```

Comment:
Knowing that I'm gonna be a pain:

Didn't we introduce a single `class` to carry all those variables?